config/configuration.json

```diff
@@ -1336,6 +1354,9 @@
       "defaultPriority": "LOW"
     }
   },
+  "ElevationUtilities": {
```

Comment:
Where is this being used? Wouldn't it cause a conflict with https://github.com/osmlab/atlas-checks/blob/fe33333d169d44c3bb679decd0c7f055d3ed27dd/config/configuration.json#L1373 ?

Reply:
Yep. I probably missed it. Even though it is only 10 lines or so down.